ServiceBusMessageForwarder.IntegrationTests/QueueTests.cs

```diff
@@ -0,0 +1,205 @@
+﻿using System;
+using System.Configuration;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.ServiceBus;
+using Microsoft.ServiceBus.Messaging;
+using ServiceBusMessageForwarder.IntegrationTests.Logging;
+using ServiceBusMessageForwarder.IntegrationTests.Models;
+using ServiceBusMessageForwarder.Logging;
+using Xbehave;
+using Xunit;
+
+namespace ServiceBusMessageForwarder.IntegrationTests
+{
+    [Collection("TestCollection")]
+    public class QueueTests
+    {
+        private readonly NamespaceManager _sourceNamespaceManager;
+        private readonly NamespaceManager _destinationNamespaceManager;
+        private readonly ILogger _logger = new MockLogger();
+        private readonly string _sourceConnectionString = ConfigurationManager.AppSettings["SourceConnectionString"];
+        private readonly string _destinationConnectionString = ConfigurationManager.AppSettings["DestinationConnectionString"];
+        private readonly string _ignoreTopics = "";
+        private readonly string _ignoreSubscriptions = "";
+        private readonly Message _testMessage = new Message { Id = 3011, Content = "Hello, World!" };
+
+        public QueueTests()
+        {
+            _sourceNamespaceManager = NamespaceManager.CreateFromConnectionString(_sourceConnectionString);
+            _destinationNamespaceManager = NamespaceManager.CreateFromConnectionString(_destinationConnectionString);
+        }
+        
+        [Scenario]
+        public void ForwardMessageWhenQueueExists()
```

Comment:
Can we say ForwardMessageWhen**Destination**QueueExists? It would be more understandable if we tell which queu we are targeting.